BeeJet.Bot/Commands/Handlers/Steam/SyncSteamCommandHandler.cs

```diff
@@ -44,18 +62,34 @@ public async Task SlashCommandExecuted()
                 gameIdMapping.Add((customId, gameChannel.Id));
                 buttonIndex++;
             }
-            
-            await Context.SlashCommandInteraction.RespondAsync("Which channels do you want to join?", ephemeral: true, components: builder.Build());
+
+            await Context.SlashCommandInteraction.RespondEphemeralAsync("Which channels do you want to join?", components: builder.Build());
             var response = await Context.SlashCommandInteraction.GetOriginalResponseAsync();
             foreach (var mapping in gameIdMapping)
             {
                 _buttonContextDb.CreateNewButtonContext(response.Id, mapping.CustomId, mapping.ChannelId.ToString());
             }
         }
 
+        private async Task<(bool IsSuccess, ulong SteamId)> TryGetSteamIdOrChallenge()
```

Comment:
Ik ben zelf niet zo van de Or in de namen, omdat het dan 2 dingen doet.
Dan zou ik eerder zeggen GetSteamId en als die niets oplevert dan LinkSteamAccount o.i.d.